changelog/unreleased/kong/fix-dns-resolv-timeout-zero.yml

```
@@ -0,0 +1,3 @@
+message: "**DNS Client**: ignore a non-positive values on resolv.conf for options timeout, and use a default value of 2 seconds instead."
```

Docs team requires that we should capitalize the first world.
 
```suggestion
message: "**DNS Client**: Ignore a non-positive values on resolv.conf for options timeout, and use a default value of 2 seconds instead."
```

---

Docs team requires that we should capitalize the first world.